__tests__/unit/storage/backup-restore.test.ts

```
@@ -0,0 +1,562 @@
+    // version がない
+    const noVersion = { ...validBackupData }
+    delete (noVersion as any).version
+    expect(validateBackupData(noVersion)).toBe(false)
+    
+    // exportDate がない
+    const noExportDate = { ...validBackupData }
+    delete (noExportDate as any).exportDate
+    expect(validateBackupData(noExportDate)).toBe(false)
+    
+    // mylists がない
+    const noMylists = { ...validBackupData }
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)